pkg/virt-controller/services/renderresources.go

```diff
@@ -404,6 +404,10 @@ func GetMemoryOverhead(vmi *v1.VirtualMachineInstance, cpuArch string, additiona
 		overhead = multiplyMemory(overhead, ratio)
 	}
 
+	if vmi.IsCPUDedicated() || vmi.WantsToHaveQOSGuaranteed() {
+		overhead.Add(resource.MustParse("100Mi"))
```

Comment:
I wonder if we should bump this to 200MiB

Main reason being: It gives the VM more space for buffers.
A lack of buffers can lead to performance issues.

But - I've got no good data to support this statement.

Reply:
The same goes for 300Mi, could be even better.
We based the 100Mi base on what we saw in our test so far. I think fine-tuning requires a more holistic approach for this.